controllers/argocd/deployment_test.go

```diff
@@ -397,6 +397,56 @@ func Test_proxyEnvVars(t *testing.T) {
 	}
 }
 
+func TestReconcileArgoCD_reconcileRepoDeployment_nodePlacement(t *testing.T) {
```

Comment:
I feel this function can look more generic.

Reply:
could you elaborate, do you mean not just _repoDeployment_ but to add other as well ?

Reply:
I feel this function makes it look that the feature is available only for repo server. Never mind, you can keep this.